CONTRIBUTING.md

```diff
@@ -1,7 +1,28 @@
-# Contributing
+# Contributing to panther-analysis
 
-Please follow the [Code of Conduct](https://github.com/panther-labs/panther-analysis/blob/main/CODE_OF_CONDUCT.md)
-in all of your interactions with the project.
+Thank you for your interest in contributing to Panther's open-source ruleset!  We value and appreciate all contributions, whether they are bug reports, feature requests, or detection rule contributions.
+
+## What Makes a Good Detection?
```

Comment:
big nit: in the docs, we use sentence case for section headers (and title case for page titles). so sentence case for this header would be: What makes a good detection?

I won't comment on all the headers on these pages, but same feedback applies